fendermint/app/src/main.rs

```diff
@@ -30,6 +32,59 @@ async fn main() {
     }
 }
 
+fn create_log(level: tracing::Level, home_dir: &Path) -> anyhow::Result<()> {
+    let log_folder = expand_tilde(home_dir)
+        .join("logs")
+        .to_str()
+        .ok_or_else(|| anyhow!("cannot parse log folder"))?
+        .to_string();
+    std::fs::create_dir_all(&log_folder).context("cannot create log folder")?;
+
+    let console_appender = config::Appender::Console;
+    let topdown_appender = config::Appender::RollingFile {
+        path: format!("{log_folder}/topdown.log"),
+        policy: Policy {
+            maximum_file_size: "10mb".to_string(),
+            // we keep the last 5 log files, older files will be deleted
+            max_size_roll_backups: 5,
+            pattern: None,
+        },
+    };
+
+    // debug level logger
+    let topdown_logger = config::Logger {
+        level: config::LevelFilter::DEBUG,
```

Comment:
Should we make this configurable too? Now we want this in debug mode for debugging purpose, but we may want to keep the same level for all in the future.

Reply:
I would say we just keep this top down logs just in case for something unexpected. The logs will be 50mb in total, it's not taking a lot of space. But we always have some logs to refer back to.